.github/workflows/interop-tests.yml

```diff
@@ -0,0 +1,27 @@
+name: Interoperability tests with GnuTLS and NSS
+on: [pull_request, push]
+jobs:
+  test:
+    runs-on: ubuntu-22.04
+    timeout-minutes: 90
+    strategy:
+      fail-fast: false
+      matrix:
+        COMPONENT: [openssl-upstream-gnutls, openssl-upstream-nss]
+    env:
+      COMPONENT: ${{ matrix.COMPONENT }}
+    # Code below inspired by https://fedoramagazine.org/github-actions-use-podman-to-run-fedora-linux/
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+      - name : Run tests
+        run: |
+          docker run --sysctl net.ipv6.conf.lo.disable_ipv6=0 -di -v /home/runner/work/openssl/openssl:/openssl --name fedora38 docker.io/nhorman/fedora38-interop:latest 
```

Comment:
As you mentioned, probably not ideal to use personal docker.io account.

How do we plan to maintain the container in future?

Reply:
why would not use preinstalled docker?

```yaml
name: Interoperability tests with GnuTLS and NSS
on: [pull_request, push]
jobs:
  test:
    runs-on: ubuntu-22.04
    timeout-minutes: 90
    strategy:
      fail-fast: false
      matrix:
        COMPONENT: [openssl-upstream-gnutls, openssl-upstream-nss]
    env:
      COMPONENT: ${{ matrix.COMPONENT }}
    container: docker.io/nhorman/fedora38-interop:latest 
    steps:
      - uses: actions/checkout@v4
        with:
          repo-token: ${{ secrets.GITHUB_TOKEN }}
      - name : Run tests
        run: |
          ln -s /interop . && ls -l ./interop/ && ./config --banner=Configured no-makedepend enable-fips --strict-warnings && perl configdata.pm --dump
          make -s -j4
          echo "Discovering tests"
          tmt run plans -n $COMPONENT discover -v
          echo "Running tests"
          tmt run -a plans -n $COMPONENT provision -h local execute -h tmt --interactive
          echo "Finished - important to prevent unwanted output truncating"
```

Reply:
@quarckster If you look at the subsequent commits, thats been removed

Reply:
You don't have to execute `docker` command. If you specify `container` on a job the context will be inside the container.

Reply:
ack I've updated the yaml to use the syntax you suggested, that looks much nicer!